### PR TITLE
Feature/cleanup config

### PR DESCRIPTION
**Pull request by insacuri**
## What? and Why?
There are a number of configurable items that don't change between environments and are likely to never change. Hard coding these in the settings file will make configuration management simpler by having a smaller list of items to manage.

NOTE: Don't merge until the config branch of the same name is also merged.

## Checklist
  - [x] CHANGELOG.md updated? (if required)

### CODE REVIEW

File: app/settings.py [hunk 1]
```diff
@@ -17,10 +17,10 @@
 
 RABBIT_QUEUE = os.getenv('RABBIT_SURVEY_QUEUE', 'survey')
 RABBIT_QUARANTINE_QUEUE = os.getenv('RABBIT_QUARANTINE_QUEUE', 'survey_quarantine')
-RABBIT_EXCHANGE = os.getenv('RABBITMQ_EXCHANGE', 'message')
+RABBIT_EXCHANGE = 'message'
 
-RABBIT_RRM_RECEIPT_QUEUE = os.getenv('RECEIPT_RRM_QUEUE', 'rrm_receipt')
-RABBIT_CTP_RECEIPT_QUEUE = os.getenv('RECEIPT_CTP_QUEUE', 'ctp_receipt')
+RABBIT_RRM_RECEIPT_QUEUE = 'rrm_receipt'
+RABBIT_CTP_RECEIPT_QUEUE = 'ctp_receipt'
```
Comment (dcdarrell9) [inline app/settings.py]: Should we keep the ctp ones for now?

Comment (insacuri) [reply app/settings.py]: Removed CTP variables

File: app/settings.py [hunk 2]
```diff
@@ -44,10 +43,7 @@
 
 RABBIT_URLS = [RABBIT_URL, RABBIT_URL2]
 
-RABBIT_CTP_QUEUE = os.getenv('SDX_COLLECT_RABBIT_CTP_QUEUE',
-                             'sdx-ctp-survey-notifications')
-RABBIT_SURVEY_QUEUE = os.getenv('SDX_COLLECT_RABBIT_QUEUE',
-                                'sdx-survey-notification-durable')
+RABBIT_SURVEY_QUEUE = 'sdx-survey-notification-durable'
```
Comment (warrenbailey) [inline app/settings.py]: Lets rename this variable to what it actually is, i.e. survey-notification

Comment (warrenbailey) [reply app/settings.py]: As discussed we'll do this in a separate PR

**Pull request merged**